android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -142,6 +152,17 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             public void afterTextChanged(Editable s) {
                 doneButton.setImageResource(R.drawable.ic_done_grey);
                 JsonFormFragmentPresenter.validate(widgetArgs.getFormFragment(), referenceEditText, false);
+                JSONObject repeatingObject = widgetArgs.getJsonObject();
+                if(repeatingObject.has(JsonFormConstants.TEXT)) {
```

Comment:
Why is this check required?

Reply:
For repeating group, If you need to know selected repeating group count value, add text field in JSON form. It is necessary to check if repeating the group contains the TEXT field so that it shouldn't update the values field. According to write Values working.
e.g
`{
        "key": "dips",
        "type": "repeating_group",
        "text": "", ...`
The text value will gets updated with user selected count. So if s/he needs to make any decision based on that.

Reply:
Also, the approach used in most of the library is :

```
String repeatingGroupCount = repeatingObject.optString(JsonFormConstants.TEXT);
if (repeatingGroupCount  != null) {
       ...
}
```

instead of using `repeatingObject.has(JsonFormConstants.TEXT)` and `repeatingObject.get(JsonFormConstants.TEXT)`. Better to use the same for consistency.